doc/source/whatsnew/v1.5.0.rst

```diff
@@ -872,6 +872,7 @@ I/O
 - Bug in :func:`read_sas` returned ``None`` rather than an empty DataFrame for SAS7BDAT files with zero rows (:issue:`18198`)
 - Bug in :class:`StataWriter` where value labels were always written with default encoding (:issue:`46750`)
 - Bug in :class:`StataWriterUTF8` where some valid characters were removed from variable names (:issue:`47276`)
+- Bug in :meth:`to_excel` when writing an empty dataframe with :class:`MultiIndex` (:issue:`19543`)
```

Comment:
DataFrame.to_excel please

Please don't resolve my remarks when I haven't checked them over. Normally the author of the remarks clicks resolve

Reply:
🙈 done